### ISSUE TITLE
not able to overwrite redux-form destroyOnUnmount default

### ISSUE THREAD

**Issue by suchostone** [redux-autoform/redux-autoform#56]
Hi! when setting destroyOnUnmount false (default is true in redux-form) it does not get updated. Seems redux-auto-form is not taking in account this prop.

Thanks.

**Comment by andrerpena** [comment]
@suchostone, this is because Redux-Autoform doesn't expect this prop. We should accept all props that Redux-Form accepts and just repass them. My fault. Sorry.

I'll take this problem this weekend.

Meanwhile, if you want to contribute and file a PR, all you have to do is to add new props to the `AutoForm` component and **repass** them to `AutoFormInternal`. I believe every property you pass to `AutoFormInternal` will automatically be received by Redux-Form.

The `AutoForm` component is quite straightforward: https://github.com/redux-autoform/redux-autoform/blob/master/src/AutoForm.js

Again.. If you can wait, I'll take a look at it this weekend. About the file upload component you asked. I just decided to prioritize the documentation flaws we got today, and right after I'll take that too.

**Comment by JonatanSalas** [comment]
@andrerpena I will take it now!

**Comment by andrerpena** [comment]
Thanks @jony. If you could see what other props are missing and add those as well that would be amazing. Thanks

**Comment by JonatanSalas** [comment]
I have added a class that performs a save of props as a workaround to extract the properties from render method and pass them to the reduxForm decorator function. Also, I have added the prop reduxFormProps which expects an object containing redux form props. 

now you should be able to do the following: 

``` jsx
render() {
     let { autoFormProps } = this.props; 
     let reduxFormProps = { 
          destroyOnUnmount: true
     };

     return <Autoform reduxFormProps={reduxFormProps} {...autoFormProps}/>
}
```

I haven't tested yet, but I expect this to be working. Tomorrow I will test it.

**PD: I will have to add validation to PropStore class in order to prevent saving props that reduxForm doesn't not recognize.**

**Comment by andrerpena** [comment]
@JonatanSalas , thanks for your quick response and your effort. I made my considerations on your commit [here](https://github.com/redux-autoform/redux-autoform/commit/5e6c7f099a43baf594073e8a7accd9cb27505926).

**Comment by andrerpena** [comment]
So @JonatanSalas , you removed the `PropStore`, which is the right thing to do, but still we didn't solve the subject of this issue. I'll address it today

**Comment by JonatanSalas** [comment]
@andrerpena have you looked at it?

**Comment by andrerpena** [comment]
As you can see [here](https://github.com/redux-autoform/redux-autoform/commit/8ba48838cb5a18a89605d5b4ae3429d3bb76c770), I added all the [props from Redux-Form](http://redux-form.com/5.3.1/#/api/reduxForm?_k=dic8ys) that I thought was safe, including the requested `destroyOnUnmount`.

However, because the bootstrap demo has been removed from RAF, I wasn't really able to test it, that's why I published it as beta.

**In order to test it, install 0.9.0-beta**.

@JonatanSalas , we have to figure out how to test it, maybe Enzyme is the way to go, but I'm not really sure how convenient that is. Something to think about.

**Comment by andrerpena** [comment]
I'll leave this issue open until I finally have time to test it.

**Comment by andrerpena** [comment]
Should be fixed with the version 0.10.0

**Comment by suchostone** [comment]
it works! thanks!
